sql/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -1253,6 +1258,21 @@ public void testAlterTableAddColumnWithCheckConstraint() throws Exception {
                      Maps.getByPath(mapping, Arrays.asList("_meta", "check_constraints", "bazinga_check")));
     }
 
+    @Test
+    public void testAlterTableDropCheckConstraint() throws Exception {
```

Comment:
Please rename the method to explain the scenario a bit better. 

```suggestion
    public void test_constraint_is_removed_after_alter_table_drop_constraint() throws Exception {
```

Or something like that.